README.md

```diff
@@ -56,18 +57,16 @@
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+> Features of the application.
+- **CSS**
+- **HTML5**
```

Comment:
:orange_circle: [OPTIONAL] Kindly add meaningful features . Rather than HTML or CSS as features, You would say as an example
-  Set up Linters
-  Add text to the HTML file
- Apply CSS style to HTML element